db/dcrpg/pgblockchain.go

```diff
@@ -3465,6 +3581,13 @@ func (pgb *ChainDB) StoreBlock(msgBlock *wire.MsgBlock, winningTickets []string,
 		// Update the best block in the meta table.
 		if err = pgb.SetDBBestBlock(); err != nil {
 			err = fmt.Errorf("SetDBBestBlock: %v", err)
+			return
+		}
+
+		// Insert the block stats. The tpi (TicketPoolInfo) should only be nil for a
+		// mainchain block if storing the genesis block.
```

Comment:
Does that mean this will never be called for the genesis block, or is that avoided elsewhere?

Reply:
Hmm. I don't know where I got that from. The `TicketPoolInfo` is not `nil` for the genesis block.